Makefile

```
@@ -0,0 +1,21 @@
+# Makefile for use as a subproject of pokeemerald, et al
+# The enforced contract is to provide a default target to compile the tool and a
+# "clean" target to sweep everything up
+
+.PHONY: all clean
+
+TARGET := poryscript
```

This all looks great, except I think the following should be added to support Windows:
```suggestion
TARGET := poryscript
ifeq ($(OS),Windows_NT)
    TARGET := $(TARGET).exe
endif
```

---

Good catch!